basic_integral/basic_integral.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+int main() {
+	int N = 10, b, i;
+	double sum, fx;
+	double p = 2*acos(0.0);
+	
+	cout << "How many iterations?" ;
+	cin >> b ;
+	
+	N = 10 * pow(b-1,2);
+	
+	for(int i=0; i<N; i++){
+		fx = sin( (2*p*i/N) + p/N )*(2*p/N);

```

Comment:
it could be a bit cleaner. Imagine someone has to read and understand your code :) why don't you define a variable delta? (and don't forget spaces)